parser.c

```diff
@@ -166,6 +166,13 @@ int parseConfig(App *app, Config *cfg)
   }
 	lwpLog(LOG_INFO, "	smooth: %f", cfg->smooth);
 
+  if (!findLine(f, "target_fps", TYPE_INT, &cfg->targetFPS))
+  {
+    lwpLog(LOG_INFO, "Can't find line 'target_fps' in config, setting to default value");
+    cfg->targetFPS = 60;
+  }
+	lwpLog(LOG_INFO, "	target_fps: %f", cfg->targetFPS);
```

Comment:
You printed it as a float 😅 
![image](https://user-images.githubusercontent.com/38699473/222924115-230c1ef7-e285-450d-9d72-44e87dbda88d.png)
Change it to `target_fps: %d`